labs/open-enterprise-agent.md

```diff
@@ -0,0 +1,97 @@
+---
+layout: default
+title: Open Enterprise Agent
+parent: Labs
+---
+# Open Enterprise Agent
+# Short Description
+The Open Enterprise Agent is a software agent for building decentralised identity applications and services in line with W3C standards and Hyperledger Aries RFCs. Aiming to be blockchain agnostic and interoperable across two key technology ecosystems in Self Sovereign Identity (SSI), it provides a scalable, robust, performant and user-friendly platform for building solutions and products based on it. Its REST API enables easy integration with any programming language, on top of a core built in Scala, a language well suited for building fault-tolerant, highly concurrent systems.
+
+# Scope of Lab
+The Open Enterprise Agent leverages the power of decentralized identity by supporting W3C standards, DIDComm v2, and Aries RFCs. It provides all the necessary features to ensure broad interoperability with SSI and web3 ecosystems. Notable high-level features of the Open Enterprise Agent include:
+
+- Ability to develop controllers using the Open Enterprise Agent API to implement business logic tailored to specific use cases.
+- Flexibility to develop controllers in any programming language.
+- Written in Scala, a powerful, object-oriented language with functional programming language features that are highly scalable, based on JVM
+- Support of did:prism, did:peer and other did-methods via a universal resolver
+- Support of JWT-VCs and Anoncreds credential formats
+- The Open Enterprise Node interfaces with the Cardano blockchain when required, ensuring high levels of security, scalability, and availability.
+- The PRISM Open Enterprise Agent is designed as a cloudagent that can be accessed from anywhere.
+
+In summary, the Open Enterprise Agent is a versatile and powerful software agent that provides a straightforward REST API for easy integration and supports industry-standard protocols, making it well-suited for a wide range of decentralized identity use cases.
+
+## Current Features
+
+### PLATFORM SUPPORT
+
+- Server
```

Comment:
Q: What does server mean here ? Does it mean Virtual machine or bare metal server or anything else ? Could you be more specific.

Reply:
Thank you for your question - and thank you for your patience with our response! 

The agent is a JVM based application (Scala) and when run in our development infrastructure - or running locally using utiltiies in the repo - is containerised using docker. We use cross-platform builds in our CI pipelines such that images are available for Linux and Mac (Intel and Apple Silicon) - I'll add a commit to this pull request to detail these and expand on this section.